docs/reviewing.md

```diff
@@ -5,7 +5,7 @@ Whilst viewing a file in an Airlock release request, they have the option of:
 * **Reject** — output is not currently acceptable for release. 
```

Comment:
I think this text here needs updating as well

Reply:
yep done